src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment.component.ts

```diff
@@ -353,7 +354,7 @@ export class FileUploadAssessmentComponent implements OnInit, OnDestroy {
                 this.complaint = res.body;
```

Comment:
A few lines up in 316 theres another error

Reply:
That method uses a more specific custom error message, so I don't want to change it.